java-manta-client-unshaded/src/main/java/com/joyent/manta/client/MantaObjectInputStream.java

```diff
@@ -245,24 +275,16 @@ public boolean markSupported() {
      * <p>If the backing stream of the connection is not a
      * {@link EofSensorInputStream}, this method with call {@link #close()}.</p>
      *
-     * <p>This method is deprecated because we can't relay on the underlying
+     * <p>This method is deprecated because we can't rely on the underlying
```

Comment:
Should this comment still say the method is deprecated?

Reply:
Noted and refactored